modules/rosa-hcp-vpc-manual.adoc

```diff
@@ -23,8 +23,14 @@ If you choose to manually create your Virtual Private Cloud (VPC) instead of usi
 | You need one availability zone for a single zone, and you need three for availability zones for multi-zone.
 
 | Public subnet
-| You must have one public subnet with a NAT gateway.
+| You must have one public subnet with a NAT gateway for public cluster. Private clusters do not need a public subnet.
 
 | DNS hostname and resolution
 | You must ensure that the DNS hostname and resolution are enabled.
+
+| AWS resource tags
+a| You must tag your subnets with the following values, based on if you are using public or private subnets:
```

Comment:
What's up with the `a`?

Reply:
I found this one recently. If you use Asciidoc syntax in a table, it is ignored unless you add the `a` in front of it:
![image](https://github.com/openshift/openshift-docs/assets/16167833/d9b514b7-8871-4dcd-add0-1a56a2b53d17)